apps/dashboard/test/system/jobs_app_test.rb

```diff
@@ -439,6 +440,7 @@ def add_bc_num_hours(project_id, script_id)
             min: 20
             step: 1
             value: '42'
+            fixed: true
```

Comment:
Thanks for updating the tests! It seems when I did this, it was serializing it as a string. Good to see it's a boolean.